vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -551,10 +551,12 @@ - (void)fetchGrants:(const std::string &)lang paymentId:(const std::string &)pay
   ledger->FetchGrants(lang, paymentId);
 }
 
-- (void)solveGrantCaptchWithPromotionId:(NSString *)promotionId solution:(NSString *)solution
+- (void)solveGrantCaptchWithPromotionId:(NSString *)promotionId solution:(NSString *)solution completion:(void (^)(BATResult result, ledger::GrantPtr grant))completion
```

Comment:
Method signature doesn't match between header + source (looks like header was changed to `BATGrant`, this has to be the same)